Add question on Windows (#24402)
DataDog/documentation: content/en/code_analysis/faq/_index.md

Datadog's static analyzer is available under an open source license and the code is [available on GitHub][4].

### Does the analyzer runs on Windows?

> Currently, the Datadog Static Analyzer only runs on UNIX-based system. We have an open feature request

Reviewer: ```suggestion
Currently, the Datadog Static Analyzer only runs on UNIX-based systems. We have an open feature request
```

> The Datadog Static Analyzer only runs on UNIX-based systems. An open feature request
> to support Windows can be [tracked on GitHub](https://github.com/DataDog/datadog-static-analyzer/issues/476).

Reviewer 2: ```suggestion
The Datadog Static Analyzer runs only on UNIX-based systems. An open feature request to support Windows can be [tracked on GitHub](https://github.com/DataDog/datadog-static-analyzer/issues/476).
```

Reviewer 2: small grammar nit

> to support Windows that can be [tracked on GitHub](https://github.com/DataDog/datadog-static-analyzer/issues/476).

Reviewer: My bad, forgot to remove `that` here.
```suggestion
to support Windows can be [tracked on GitHub](https://github.com/DataDog/datadog-static-analyzer/issues/476).
```

> to support Windows that is [trackable on GitHub](https://github.com/DataDog/datadog-static-analyzer/issues/476).

Reviewer: ```suggestion
to support Windows that can be [tracked on GitHub](https://github.com/DataDog/datadog-static-analyzer/issues/476).
```
